theme.json

```diff
@@ -57,38 +59,64 @@
 		"typography": {
 			"fontFamilies": [
 				{
-					"fontFamily": "-apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen-Sans, Ubuntu, Cantarell, 'Helvetica Neue', sans-serif",
-					"name": "System Font",
-					"slug": "system-font"
-				}
+					"fontFace": [
+						{
+							"fontDisplay": "block",
+							"fontFamily": "league_gothicregular",
+							"fontStretch": "normal",
+							"fontStyle": "normal",
+							"fontWeight": "200 800",
+							"src": [
+								"file:./assets/fonts/leaguegothic-regular-variablefont_wdth-webfont.woff2"
+							]
+						}
+					],
+					"fontFamily": "\"league_gothicregular\", serif",
+					"name": "League Gothic",
+					"slug": "league-lothic"
+				},
+        {
+          "fontFace": [
+            {
+							"fontDisplay": "block",
+							"fontFamily": "EB Garamond",
+							"fontStretch": "normal",
+							"fontStyle": "normal",
+							"fontWeight": "200 800",
+							"src": [
+								"file:./assets/fonts/ebgaramond-regular.woff2"
+							]
+            }
+          ],
+          "fontFamily": "\"EB Garamond\", serif",
+					"name": "EB Garamond",
+					"slug": "eb-garamond"
+        }
 			],
 			"fontSizes": [
 				{
-					"name": "Small",
-					"size": "0.825rem",
-					"slug": "small"
+          "name": "Extra small",
```

Comment:
Would you be able to set your editor config so that it doesn't use a mix of spaces and tabs in this file? We should try to pick one and stick to it.

Reply:
Fixed on open PR #18  and #19